groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -64,9 +72,15 @@
     private static final Logger log = LoggerFactory.getLogger(GroovycWrapper.class);
 
     private static final String GROOVY_EXTENSION = "groovy";
+    private static final String GROOVY_DEFAULT_INTERFACE = "groovy.lang.GroovyObject";
+    private static final String JAVA_DEFAULT_OBJECT = "java.lang.Object";
+
     private final Path workspaceRoot;
     private final CompilationUnit unit;
+    // Maps from source file -> set of symbols in that file
     private Map<String, Set<SymbolInformation>> fileSymbols = Maps.newHashMap();
+    // Maps from type -> set of references of this type
+    private Map<String, Set<SymbolInformation>> references = Maps.newHashMap();
```

Comment:
We may want to rename this field to classReferences to be clear.
And I imagine after we have the reference information of other types (like local variables), it might still be easier to keep them in different variables